packages/itwinui-react/src/styles.js/styles.module.css

```diff
@@ -13,3 +13,13 @@
     --_iui-v3-loaded: yes;
   }
 }
+
+@layer itwinui {
+  *,
+  *::before,
+  *::after {
+    /* When --_iui-width is set, inline-size will use it, otherwise it will be automatically reverted. */
+    --_iui-width: initial;
```

Comment:
If `--_iui-width` is initialized, will the fallback part of the `var()` (i.e. `revert-layer`) not be triggered?

Reply:
No, when a custom property is set to `initial`, it will trigger the fallback.